cmd/admin-heal-stop.go

```diff
@@ -0,0 +1,81 @@
+/*
+ * Minio Client (C) 2017 Minio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/minio/cli"
+)
+
+var adminHealStopCmd = cli.Command{
+	Name:            "stop",
+	Usage:           "Stop a heal task",
+	Before:          setGlobalsFromContext,
+	Action:          mainAdminHealStop,
+	Flags:           globalFlags,
+	HideHelpCommand: true,
+	CustomHelpTemplate: `NAME:
+  {{.HelpName}} - {{.Usage}}
+
+USAGE:
+  {{.HelpName}} [FLAGS] TARGET
+
+FLAGS:
+  {{range .VisibleFlags}}{{.}}
+  {{end}}
+EXAMPLES:
+    1. To stop a heal task with no bucket and object-prefix in a Minio server with alias 'play'
+       $ {{.HelpName}} play
+
+    2. To stop a heal task on bucket 'testbucket' in a Minio server with alias 'play'
+       $ {{.HelpName}} play/testbucket/
+
+`,
+}
+
+// mainAdminHealStop - the entry function of heal status command
+func mainAdminHealStop(ctx *cli.Context) error {
+	// Check for command syntax
+	checkAdminHealSyntax(ctx)
+
+	// Get the alias parameter from cli
+	args := ctx.Args()
+	aliasedURL := args.Get(0)
+
+	// Create a new Minio Admin Client
+	client, err := newAdminClient(aliasedURL)
+	if err != nil {
+		fatalIf(err.Trace(aliasedURL), "Cannot initialize admin client.")
+		return nil
+	}
+
+	// Compute heal path
+	aliasedURL = filepath.ToSlash(aliasedURL)
+	splits := splitStr(aliasedURL, "/", 3)
+	bucket, prefix := splits[1], splits[2]
+
+	myerr := client.HealStop(bucket, prefix)
```

Comment:
We don't need myerr and also we should use `probe` error on client side as consistently used everywhere.

Reply:
Done.